### PR TITLE
6 ➡️ 7 (main)

### PR DESCRIPTION
**Pull request by chapulina**
# ➡️ Forward port

* Port  `ign-gazebo6` to `main`
* Branch comparison: https://github.com/ignitionrobotics/ign-gazebo/compare/main...ign-gazebo6
* Requires https://github.com/ignitionrobotics/ign-sensors/pull/191

**Note to maintainers**: Remember to **Merge** with commit (not squash-merge or rebase)

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -6,6 +6,72 @@
 
 ### Ignition Gazebo 6.X.X (202X-XX-XX)
 
+### Ignition Gazebo 6.4.0 (2021-01-13)
```
Comment (j-rivero) [inline Changelog.md:9]: I see `ign-math6` branch on 6.9.3 version. Do we need to sync this or the PR is not related to the latest state of `ign-math`?

Comment (chapulina) [reply Changelog.md:9]: This PR is targeting Garden, which will use `ign-math7` (i.e. `main`). This 6.4.0 here is Gazebo's changelog

**Comment by j-rivero** [comment]
CMake warnings in Ubuntu and Windows, probably out of the scope of this PR. I think that the failing test in Ubuntu is also unrelated.

**Comment by chapulina** [comment]
> CMake warnings in Ubuntu and Windows, probably out of the scope of this PR.

Yeah the Ubuntu ones should be fixed by ign-gui releases.

Windows is in the works, i.e. https://github.com/ignitionrobotics/ign-gazebo/pull/1299

> I think that the failing test in Ubuntu is also unrelated.

Yup, that's been flaky

File: Changelog.md (hunk 1 above)
Comment (j-rivero) [reply Changelog.md:9]: aaaaaah! Sorry for the confusion.

**Review by j-rivero — APPROVED** [review]
I can not detect problems in the merg or missing parts. If CI is happy, I'm happy too.

**Pull request merged**